config/clusters/catalystproject-africa/uvri.values.yaml

```diff
@@ -0,0 +1,70 @@
+jupyterhub:
+  ingress:
+    hosts: [uvri.af.catalystproject.2i2c.cloud]
+    tls:
+      - hosts: [uvri.af.catalystproject.2i2c.cloud]
+        secretName: https-auto-tls
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
+        org:
+          name: Catalyst Project, Africa - UVRI
+          url: https://2i2c.org
+          logo_url: https://2i2c.org/media/logo.png
+        designed_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        operated_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        funded_by:
+          name: Chan Zuckerberg Initiative - Open Science
+          url: "https://chanzuckerberg.com/science/programs-resources/open-science/"
+  hub:
+    config:
+      JupyterHub:
+        authenticator_class: github
+      GitHubOAuthenticator:
+        oauth_callback_url: https://uvri.af.catalystproject.2i2c.cloud/hub/oauth_callback
+        allowed_organizations:
+          - CatalystProject-Hubs:uvri
+        scope:
+          - read:org
+      # Authenticator:
+      #   admin_users:
+      #     -
+  singleuser:
+    profileList:
+      - display_name: Jupyter
+        slug: jupyter-scipy
+        description: "Python environment"
+        default: true
+        kubespawner_override:
+          image: jupyter/scipy-notebook:2023-06-27
+          default_url: /lab
+        profile_options: &profile_options
+          resource_allocation:
+            display_name: Resource Allocation
+            choices:
```

Comment:
I think using choices here is overkill if you are only providing one option, you could probably just merge the `kubespawner_override` section below here with the one above that sets the image (same in the R profile as well)

Reply:
I agree with @sgibson91 about not having `profile_options` in this case. 

Also, @AIDEA775, if you want to get more insight into configuring profile lists, you can checkout https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html#kubespawner.KubeSpawner.profile_list.

Reply:
I'd advocate that we provide this with multiple options, defaulting to a lower request in https://github.com/2i2c-org/infrastructure/issues/3631#issuecomment-1905365573.

Ideally we'd have lots of guidance about this and discussed requests/limits already, but we haven't yet found time to conclude such guidance.

This doesn't have to be a PR blocker, but I think there is UX value to provide a single option already to not introduce it later as a change.